region/lowernorfair/east.json

```diff
@@ -814,6 +845,38 @@
         {
           "from": 3,
           "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "G-Mode Deep Stuck X-Ray Climb",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [3],
+                      "mode": "direct",
+                      "artificialMorph": false
+                    }},
+                    "canXRayClimb",
+                    {"heatFrames": 1650}
+                  ]
+                },
+                {
+                  "name": "Shinespark Deep Stuck X-Ray Climb",
+                  "notable": false,
+                  "requires": [
+                    {"canComeInCharged": {
+                      "fromNode": 3,
+                      "shinesparkFrames": 0,
```

Comment:
For some reason I was thinking that it could be done with 0 frames, but you're right it seems to require at least 1. So I updated this and also went back and added it for the West Ocean deep stuck strats.